### ISSUE TITLE
Full name of contacts cannot be changed anymore

### ISSUE THREAD

**Issue by Pantek59** [owncloud/contacts#866]
When I change the first or last name of a contact, I would expect its label in the list to change accordingly. However, when I change the contact's name, the old name is still shown in the list. When I open the contact, the new name is correctly shown in the text field.

Owncloud 8.0 / Ubuntu 12.04 / Contacts 0.3.0.18

**Comment by cova-it** [comment]
Similar to my problem: changed the name in contacts app and it appeared correctly. But in the vCard itself the FN doesn't change, so the contacts with a name changed in the contacts app are displayed wrong on my smartphone.

**Comment by vayw** [comment]
Same with ownCloud 8.0.2 (stable) / Contacts 0.3.0.18

**Comment by may-bee** [comment]
I also observe this behavior having the same configuration as @vayw posted above.
Hope for a quick fix as this is a very important basic functionality.

**Comment by may-bee** [comment]
persistent in ownCloud 8.0.3 (stable)

**Comment by RealRancor** [comment]
Hi,

could this be related to https://github.com/owncloud/contacts/issues/823 which got fixed by https://github.com/owncloud/contacts/pull/850 ?

**Comment by Pantek59** [comment]
@RealRancor Yes, I do think it's related. I made the manual change of contacts.css and when I use the separate fields to edit the full name, the full name is actually changed. You have to reload the list to see the change, but it happens.
#850 fixes this problem, so I hope it will be implemented soon, so that the change does not have to be made manually.
